sentry_sdk/consts.py

```diff
@@ -49,7 +50,9 @@
             "enable_metrics": Optional[bool],
             "metrics_summary_sample_rate": Optional[float],
             "should_summarize_metric": Optional[Callable[[str, MetricTags], bool]],
-            "before_emit_metric": Optional[Callable[[str, MetricTags], bool]],
+            "before_emit_metric": Optional[
+                Callable[[str, Any, MeasurementUnit, MetricTags], bool]
```

Comment:
Can we use a narrower type than Any? Perhaps object? This also applies to the other functions in this PR typed as Any

Reply:
Changed to `MetricValue` 👍🏻